lightning_pose/apps/video_diagnostics.py

```diff
@@ -0,0 +1,202 @@
+"""Analyze predictions on video data.
+
+Users select an arbitrary number of csvs (one per model) from their file system
```

Comment:
same comment as other app, replace these docs with a reference to the markdown file